README.md

```diff
@@ -15,3 +15,8 @@ Issue content goes here
 
 /cc @twolfson
 ```
+
+## Attribution
+
+* Gear icon by [Jardson Almeida](https://thenounproject.com/search/?q=gear&i=102547)
```

Comment:
For organizational consistency, can we use `-` for bullets? (e.g. `gmusic.js's README` is using `-`)